+ source ../common_lib/fsio_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
++ export 'FS_TYPE=ext3 ext4 xfs'
++ FS_TYPE='ext3 ext4 xfs'
++ export LANG=en_US.UTF-8
++ LANG=en_US.UTF-8
++ REMOVE_VG
++ vggroup=
++ [[ '' != '' ]]
+++ df -T
+++ grep test_vggroup
+++ awk '{print $7}'
++ mounted=
++ [[ '' != '' ]]
++ lsblk
++ grep test_vggroup
++ [[ 1 -eq 0 ]]
++ rm -rf '/tmp/point*'
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Wed Aug  9 02:43:13 2023 - INFO  - Start to prepare the database config.
++ date +%Y%m%d%H%M%S
+ cur_date=20230809024313
++ CREATE_VG
+++ date +%Y%m%d%H%M%S
++ cur_date=20230809024313
+++ lsblk
+++ grep disk
+++ awk '{print $1}'
+++ tail -n 1
++ free_disk=vdb
++ disk_name=/dev/vdb
++ pvcreate /dev/vdb -f
++ [[ 5 -eq 0 ]]
+++ pvcreate /dev/vdb
+++ grep test_vggroup
+++ cut -d '"' -f 4
++ vggroup=test_vggroup20230809023817
++ printf test_vggroup20230809023817
+ vggroup=test_vggroup20230809023817
+ lv=test_lv20230809024313
+ lvcreate -n test_lv20230809024313 -L 2G test_vggroup20230809023817 -y
+ mkfs.xfs /dev/test_vggroup20230809023817/test_lv20230809024313
oe_test_FSIO_remkfs.sh: line 27: mkfs.xfs: command not found
+ point=/mnt/point20230809024313
+ mkdir /mnt/point20230809024313
+ LOG_INFO 'Finish to prepare the database config.'
+ message='Finish to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish to prepare the database config.'
Wed Aug  9 02:43:15 2023 - INFO  - Finish to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Aug  9 02:43:16 2023 - INFO  - Start to run test.
+ mkfs.xfs -f /dev/test_vggroup20230809023817/test_lv20230809024313
oe_test_FSIO_remkfs.sh: line 35: mkfs.xfs: command not found
+ mount /dev/test_vggroup20230809023817/test_lv20230809024313 /mnt/point20230809024313
mount: /mnt/point20230809024313: wrong fs type, bad option, bad superblock on /dev/mapper/test_vggroup20230809023817-test_lv20230809024313, missing codepage or helper program, or other error.
       dmesg(1) may have more information after failed mount system call.
+ df -T
+ grep /mnt/point20230809024313
+ grep xfs
+ CHECK_RESULT 1 0 0 'mkfs xfs to be ext4 failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='mkfs xfs to be ext4 failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'mkfs xfs to be ext4 failed.'
+ LOG_ERROR 'mkfs xfs to be ext4 failed.'
+ message='mkfs xfs to be ext4 failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'mkfs xfs to be ext4 failed.'
Wed Aug  9 02:43:17 2023 - ERROR - mkfs xfs to be ext4 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_FSIO_remkfs.sh line 38'
+ message='oe_test_FSIO_remkfs.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_FSIO_remkfs.sh line 38'
Wed Aug  9 02:43:18 2023 - ERROR - oe_test_FSIO_remkfs.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Wed Aug  9 02:43:19 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 1
+ test 1 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Aug  9 02:43:19 2023 - ERROR - failed to execute the case.
+ exit 1
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Aug  9 02:43:20 2023 - INFO  - Start to restore the test environment.
+ REMOVE_FS /mnt/point20230809024313
+ point_list=($1)
++ seq 0 0
+ for i in $(seq 0 $((${#point_list[@]} - 1)))
+ tmp=/mnt/point20230809024313
++ df -T
++ grep /mnt/point20230809024313
++ awk '{print $1}'
+ lv=
+ [[ '' == '' ]]
+ continue
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Wed Aug  9 02:43:21 2023 - INFO  - End to restore the test environment.
